ckanext/qdes/helpers.py

```diff
@@ -95,6 +102,8 @@ def qdes_review_datasets(org_id=None):
             org_ids.append(organization.id)
         query = query.filter(Package.owner_org.in_(org_ids))
 
+    if contact_point:
+        query = query.filter(PackageExtra.key=='contact_point').filter(PackageExtra.value==contact_point[0].get('value'))
     packages = query.all()
```

Comment:
@tino097 If the user is not in the point of contacts we should not display any datasets so return an empty list,